.github/dependabot.yml

```diff
@@ -6,7 +6,7 @@ updates:
     interval: daily
     time: "03:00"
     timezone: Europe/London
-  open-pull-requests-limit: 10
+  open-pull-requests-limit: 0
   reviewers:
```

Comment:
We can remove the reviewers. That's why we are adding codeowners files, so the PR reviewers are inferred from a GitHub team (`form-builder-pr-reviewers`)